### PR TITLE
adding my fish tank

### PR DESCRIPTION
**Pull request by Mayorman07**
Could you kindly add me to this list @royranger

### CODE REVIEW

**Review by donmarvex — APPROVED** [review]
Really great animation! Thank You @Mayorman07

File: include.js [hunk 1]
```diff
@@ -5904,8 +5904,16 @@ let cards = [
     author: 'Viskarra',
     githubLink: 'https://github.com/viskarra'
   }
```
Comment (donmarvex) [inline include.js:5906]: Please fix an error by adding a comma before your object details @Mayorman07

**Review by donmarvex — COMMENTED** [review]
Changes requested below

File: include.js (hunk 1 above)
Comment (Mayorman07) [reply include.js:5906]: this has been fixed now, thank you for pointing it out @donmarvex

Comment (donmarvex) [reply include.js:5906]: @Mayorman07 could you create another pull request with all you work

**Pull request merged**